pkgs/http/test/io/client_conformance_test.dart

```diff
@@ -10,5 +10,8 @@ import 'package:http_client_conformance_tests/http_client_conformance_tests.dart
 import 'package:test/test.dart';
 
 void main() {
-  testAll(IOClient.new);
+  testAll(IOClient.new,
+      preservesMethodCase:
+          false // https://github.com/dart-lang/sdk/issues/54187
```

Comment:
nit: perhaps move this comment up a line to get better formatting?

Reply:
I used a shorter link to fit it on the same line.